docs/installation.rst

```diff
@@ -142,6 +142,7 @@ following dependencies first:
 * OpenSSL_ >= 1.0.0, plus development headers.
 * libacl_ (which depends on libattr_), both plus development headers.
 * liblz4_, plus development headers.
+* ZeroMQ_ >= 4.0.0, plus development headers.
```

Comment:
ok course debian has cleanly split packages, but if you list pyzmq pypi package files, you'll find the lib zeromq .so.